packages/grid/x-data-grid/src/components/base/GridOverlays.tsx

```diff
@@ -8,12 +9,20 @@ import {
 } from '../../hooks/features/rows/gridRowsSelector';
 import { useGridApiContext } from '../../hooks/utils/useGridApiContext';
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
-import { gridDensityTotalHeaderHeightSelector } from '../../hooks/features/density/densitySelector';
+import { getMinimalContentHeight } from '../../hooks/features/rows/gridRowsUtils';
+
+const GridOverlayWrapperRoot = styled('div')({
```

Comment:
The `name`, `slot` and `overridesResolver` are missing. Could you add them? Same for the inner div.

Reply:
@m4theushw Done, can you take a look?

Reply:
How about `OverlayWrapper` and `OverlayWrapperInner` for the wrapper and the inner div, respectively? It's simpler.

Reply:
Sounds good 👍